tests/policies/test_policies.py

```diff
@@ -36,31 +34,37 @@ def test_actions_valid(env_name, policy_type):
         assert venv.action_space.contains(a)
 
 
-def test_save_stable_model_errors_and_warnings(tmpdir):
+@pytest.mark.parametrize(
+    "policy_env_pair",
+    [
+        ("ppo", SIMPLE_DISCRETE_ENV),
+        ("sac", SIMPLE_CONTINUOUS_ENV),
+    ],
+)
+def test_save_stable_model_errors_and_warnings(tmpdir, policy_env_pair):
     """Check errors and warnings in `save_stable_model()`."""
+    policy, env = policy_env_pair
```

Comment:
```suggestion
    policy, env_name = policy_env_pair
```
we usually use `env` to refer to a `gym.Env`

Reply:
accepted!